.github/workflows/build.yml

```
@@ -39,6 +39,31 @@ on:
         required: false
         type: string
         default: 'v1'
+      USE_DYNAMIC_IMAGE_TAG:
+        description: 'If this field is set to true, the image tag configured in the config.yml file will be skipped, and a dinamic generated image tag will be used instead.'
+        required: false
+        type: boolean
+        default: false
+      UPDATE_REMOTE_CONFIG:
+        description: 'If this field is set to true, a commit will be created with the updated image tag name in the config.yml file'
+        required: false
+        type: boolean
+        default: false
+      USE_RANDOM_SUFFIX:
```

I think the current name works fine, but leaving a suggestion just in case you find it helpful
```suggestion
      ADD_RANDOM_SUFFIX_TO_IMAGE_TAG:
```